### ISSUE TITLE
Make it work without Flash

### ISSUE THREAD

**Issue by gregra81** [asafdav/ng-clip#78]
Chrome is dropping Flash support, this will basically make this wonderful directive unusable.

**Comment by asafdav** [comment]
It's a very good idea! 

I'll try to find some time to work on this, but I would love to merge a pull request that tackles this if someone has the time.

**Comment by asafdav** [comment]
Hi @gregra81, I started to work on a new version that doesn't require flash, it would be very helpful if you could check this branch [clipboard-js](https://github.com/asafdav/ng-clip/tree/clipboard-js) and let me know if it works for you. 

Thanks!

**Comment by gregra81** [comment]
OK, @asafdav, I'll take a look as soon as I have time. BTW, I thought that copying from your browser to the computer via JS poses a security threat and browsers don't allow it. Am I wrong?

**Comment by gregra81** [comment]
@asafdav, I gave it a quick check. First of all, thank you for your time in developing that. 
For now, it works well on Chrome. Is there a plan to push it to other browsers?
Is there a plan releasing it as a stable version?

**Comment by asafdav** [comment]
Hi @gregra81, yes I hope to release it soon. I'll have to implement the missing features. thanks for taking the time and help my out.

**Comment by maslowivan** [comment]
Do it please if you still support this repo

**Comment by maslowivan** [comment]
Or do some another bower package

**Comment by niemyjski** [comment]
https://github.com/asafdav/ng-clip/issues/88

**Comment by niemyjski** [comment]
Yeah I think it would be great to use the execAction api and then if that doesn't work we make a request to load the flash version and try it..

**Comment by niemyjski** [comment]
@asafdav I see your branch by why even have the dependency on clipboard js.. It's super simple to just call execAction('copy') and wrap that in a try block.

**Comment by gregra81** [comment]
@niemyjski - I suggest you fork the repo, add this and make a pull request. What do you think?

**Comment by niemyjski** [comment]
I already switched to angular-clipboard which uses this api
